.github/workflows/prepare-for-prod-deploy.yml

```diff
@@ -135,3 +135,20 @@ jobs:
           git push -u origin shared-transformer-$TAG_NAME
 
           hub pull-request -m "chore: upgrade shared transformers to $TAG_NAME"
+
+      - name: Update Helm Chart and Raise Pull Request For Hosted Transformer
```

Comment:
Why a separate pull request?

Reply:
Usually, we do not deploy changes on the hosted transformer for every release, especially when there is a custom build running on it. In such cases, having a separate pull request can help us make a decision on whether we should deploy the changes on the hosted transformer as well.